source/_dashboards/entities.markdown

```diff
@@ -85,7 +85,7 @@ secondary_info:
   type: string
 format:
   required: false
-  description: "How the state should be formatted. Currently only used for timestamp sensors. Valid values are: `relative`, `total`, `date`, `time` and `datetime`."
+  description: "How the state should be formatted. Currently only used for datetime sensors with `device_class: timestamp`. Valid values are: `relative`, `total`, `date`, `time` and `datetime`."
```

Comment:
This seems incorrect. The existing wording is right.
We don't have datetime sensors (not yet, that is actually in process). The device class already defines what type of entity (in this case sensor), thus  the existing "timestamp sensors" term is correct.

../Frenck

Reply:
`device_class: timestamp` <- is a technical detail on the classification looks like in code. This is normal text.